test/create-stream-vs-put-racecondition.js

```diff
@@ -3,7 +3,7 @@
  * MIT License <https://github.com/level/levelup/blob/master/LICENSE.md>
  */
 
-var levelup = require('../lib/levelup.js')
+var levelup = require('../lib/levelup.js').default
```

Comment:
I believe this is not necessary because we are both had exported `LevelUP` and `default`.

See: https://github.com/Level/levelup/blob/f93790fc9d296d6be0691d52687f0c204f116473/lib/levelup.js#L294

That will make sure the code will pass the following assertion:
```
const levelup1 = require('../lib/levelup.js')
const levelup2 = require('../lib/levelup.js').default

assert(levelup1 === levelup2)
```

Reply:
Its down to typescript - you cannot have both default and `export = ` in the typings. So the typings have gone with the `default` export exclusively. It means when you run the normal js through the typescript compiler it will complain if you don't use the default.

Reply:
Ok, understand.

Then how about we do a `module.exports = LevelUP.default = LevelUP.LevelUP = Levelup` ?

Then we will be able to do a `var { LevelUP } = require('../lib/levelup.js')`, which would be better than `require('...').default`?

Reply:
I'm not a fan of `require('...').default` - but because it's only in the tests, I prefer it over more hacks in `lib/levelup.js`.

Reply:
yeah I don't think we need more hacks to the export; Its just for the test here because we're pushing the (mostly) unadulterated js through the compiler. 

you can always do const `{default:levelup} =require('levelup');` but I suspect we might run into issues if we're still testing on an older node.

Reply:
I'm ok with `default` too because it's just unit test script.